doc/sphinxman/source/scf.rst

```diff
@@ -528,6 +528,36 @@ in a 3-21G basis and then casting up to cc-pVTZ is shown below::
 
     energy('scf')
 
+.. .. _`scf:restart`:
+
+Restarting the SCF
+~~~~~~~~~~~~~~~~~~
+
+Reading orbital data from a previous calculations is done via the ``restart_file`` option,
+where the actual file is a serialized ``wfn`` object (see :ref:`saving the wfn <sec:save_wfn>` )
+By default the orbital data file of the converged SCF(``psi.PID.name.180.npy``) is deleted after |PSIfour| exists or the ``clean()`` function is called.
+To keep the orbital data and also write the orbitals after every iteration the ``write_orbitals`` options is available: ::
+
+  energy('scf', write_orbitals='my_mos'),
+
+which write a ``Wavefunction`` object converted (serialized) to a numpy file called ``my_mos.npy``.
+The restart can then be done as follows: ::
+
+  set guess read
+  energy('scf', restart_file='my_mos')
+
+The ``.npy`` ending is optional.
```

Comment:
Specifying the ``.npy`` suffix when writing and reading restart files is optional.